kube-runtime/src/reflector/store.rs

```diff
@@ -219,4 +237,30 @@ mod tests {
         let store = store_w.as_reader();
         assert_eq!(store.get(&ObjectRef::from_obj(&nsed_cm)).as_deref(), Some(&cm));
     }
+
+    #[test]
+    fn find_element_in_store() {
+        let cm = ConfigMap {
+            metadata: ObjectMeta {
+                name: Some("obj".to_string()),
+                namespace: None,
+                ..ObjectMeta::default()
+            },
+            ..ConfigMap::default()
+        };
+        let mut target_cm = cm.clone();
+        let mut store_w = Writer::default();
+        store_w.apply_watcher_event(&watcher::Event::Applied(cm));
+
+        let store = store_w.as_reader();
```

Comment:
I think this can re-use the pair constructor for the store as a minor simplification

```suggestion
        let (mut writer, reader) = store::<ConfigMap>();
        writer.apply_watcher_event(&watcher::Event::Applied(cm));
```

Reply:
done